WordPress/WordPressTest/ReaderSubscribeCommentsActionTests.swift

```diff
@@ -88,8 +88,8 @@ private class MockFollowCommentsService: FollowCommentsService {
         super.init(post: post)
     }
 
-    @objc required init?(post: ReaderPost, remote: ReaderPostServiceRemote = ReaderPostServiceRemote.withDefaultApi()) {
-        super.init(post: post, remote: remote)
+    required init?(post: ReaderPost, coreDataStack: CoreDataStack, remote: ReaderPostServiceRemote) {
+        fatalError("unsupported initialiser")
```

Comment:
I believe this was here just because compiler says so, it's not meant to be used in the unit tests. So, I changed it to throw a fatal error to make this expectation explicit.

Reply:
Fair 👍